src/sentry/static/sentry/app/actionCreators/members.jsx

```diff
@@ -14,6 +14,22 @@ export function fetchOrgMembers(api, orgId) {
   });
 }
 
+/**
+ * Convert a list of members with user & project data
+ * into a object that maps project slugs : users in that project.
+ */
+export function indexMembersByProject(members) {
```

Comment:
I see this as more of a utility function rather than an action. It'd be good to have a definition of what is considered an action in our frontend TSC.

Reply:
👍